earthpy/plot.py

```diff
@@ -70,33 +71,34 @@ def colorbar(mapobj, size="3%", pad=0.09, aspect=20):
     return fig.colorbar(mapobj, cax=cax)
 
 
-# Function to plot all layers in a stack
 def plot_bands(
     arr, title=None, cmap="Greys_r", figsize=(12, 12), cols=3, extent=None
 ):
-    """Plot each layer in a raster stack read from rasterio in
-    (band, row , col) order as a numpy array. plot_bands will create an
-    individual plot for each band in a grid.
+    """Plot each layer in a raster stack read from rasterio as a numpy array.
+    plot_bands will create an individual plot for each band in a grid.
 
     Parameters
     ----------
-    arr: numpy array
-        An n-dimensional numpy array with the order and numpy shape: (nbands, nrows, ncols)
-    title: str or list
-        Title of one band, or list of titles with one title per band
-    cmap: str
-        Colormap name ("greys" by default)
-    cols: int
-        Number of columns for plot grid (default: 3)
-    figsize: tuple - optional
+    arr : numpy array
+        An n-dimensional numpy array to plot.
+    title : str or list (optional)
+        Title of one band or list of titles with one title per band.
+    cmap : str (optional)
+        Colormap name for plots.
+    cols : int (optional)
+        Number of columns for plot grid.
+    figsize : tuple (optional)
```

Comment:
Same comment here - you have removed important information from the docstring. Please keep that information as it is. The cmap for example is not optional, it just has a defaule value, similarly cols is the same. please revert this to what was there before.